packages/rpc/test/marshal.test.ts

```diff
@@ -0,0 +1,11 @@
+import '@endo/init';
```

Comment:
consider noting,
```suggestion
import '@endo/init'; // UNTIL https://github.com/endojs/endo/issues/1686
```

Reply:
I don't recognize that one by number... let's see...

 - https://github.com/endojs/endo/issues/1686

I don't understand how marshal's `fromCapData` would correctly build immutable data without `@endo/init`.

I guess I'm not clear on the Hardened JS norms in ui-kit either, so ok.

Reply:
This test doesn't need to cover mutability. Once 1686 is resolved, this test could check the string without hardening.